app/services/sources/ImgurSource.java

```diff
@@ -103,8 +106,8 @@ public long getQueryDelta() {
 
     private Post createPost(JsonObject object) {
         Post.Builder builder = Post.newBuilder();
-        builder.setId(object.getAsJsonPrimitive("id").getAsString());
-        builder.setTimestamp(object.getAsJsonPrimitive("datetime").getAsLong());
+        builder.setId(object.getAsJsonPrimitive("id").getAsString());           // convert from seconds to millis
```

Comment:
I disagree, I find it very confusing to have a comment on a line for which it does not pertain.